py/test/unit/selenium/webdriver/remote/remote_connection_tests.py

```diff
@@ -24,6 +25,28 @@
 from selenium.webdriver.remote.remote_connection import RemoteConnection
 
 
+@pytest.fixture
+def remote_connection():
+    return RemoteConnection("http://localhost:4444")
+
+
+def test_add_command():
+    RemoteConnection.add_command("CUSTOM_COMMAND", "PUT", "/session/$sessionId/custom")
+    assert RemoteConnection.extra_commands["CUSTOM_COMMAND"] == ("PUT", "/session/$sessionId/custom")
+
+
+@patch("selenium.webdriver.remote.remote_connection.RemoteConnection._request")
+def test_execute_custom_command(mock_request, remote_connection):
+    RemoteConnection.add_command("CUSTOM_COMMAND", "PUT", "/session/$sessionId/custom")
+    mock_request.return_value = {"status": 0, "value": "OK"}
```

Comment:
I don't know appium... but why are we using `status` rather than HTTP codes?

Reply:
I believe both are the same here.

Reply:
Both seems similar, do you want me to use `status_code`?

Reply:
Changed status to `200` in recent commit, thanks for pointing out!